pkg/agent/flowexporter/types.go

```diff
@@ -51,19 +51,26 @@ type Connection struct {
 	Labels, LabelsMask []byte
 	// TODO: Have a separate field for protocol. No need to keep it in Tuple.
 	TupleOrig, TupleReply          Tuple
+	FlowKey                        Tuple
 	OriginalPackets, OriginalBytes uint64
 	ReversePackets, ReverseBytes   uint64
 	// Fields specific to Antrea
-	SourcePodNamespace            string
-	SourcePodName                 string
-	DestinationPodNamespace       string
-	DestinationPodName            string
-	DestinationServicePortName    string
-	IngressNetworkPolicyName      string
-	IngressNetworkPolicyNamespace string
-	EgressNetworkPolicyName       string
-	EgressNetworkPolicyNamespace  string
-	TCPState                      string
+	SourcePodNamespace             string
+	SourcePodName                  string
+	DestinationPodNamespace        string
+	DestinationPodName             string
+	DestinationServicePortName     string
+	IngressNetworkPolicyName       string
+	IngressNetworkPolicyNamespace  string
+	IngressNetworkPolicyRuleAction uint8
+	EgressNetworkPolicyName        string
+	EgressNetworkPolicyNamespace   string
+	EgressNetworkPolicyRuleAction  uint8
+	TCPState                       string
+	// fields specific to deny connections
+	Bytes    uint64
```

Comment:
maybe add a comment explaining what this field is for. I am not sure why we can't use `OriginalPackets` / `OriginalBytes`? Also, what do we use for the packet count?

Reply:
OriginalPackets/OriginalBytes are total count. What we can obtain from the packetin message is the packet length of one packet. Added DeltaBytes, DeltaPackets, TotalBytes, TotalPackets for us to keep track.